# Attitude States
float64 roll
float64 pitch
float64 yaw
float64 rolld
float64 pitchd
float64 yawd
float64 rolldd
float64 pitchdd
float64 yawdd
